package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A hapi plugin integrating Objection.js ORM",
   "main": "lib/index.js",
   "scripts": {
-    "test": "NODE_ENV=test lab -a code -t 100 -L -I '__core-js_shared__'",
+    "test": "NODE_ENV=test lab --timeout 4000 -a code -t 100 -L -I '__core-js_shared__'",
```

Comment:
As for `__core-js_shared__`, this is indeed an artifact of using babel.  We shouldn't need to use babel, so we oughtta figure out what's going on there.